src/test/skript/tests/syntaxes/expressions/ExprUUID.sk

```diff
@@ -0,0 +1,31 @@
+test "uuid":
```

Comment:
I would like to see more tests for possible breaking changes
Mainly I am worried about:
```
set {_uuid} to entity's uuid
set {_entity} to {_uuid} parsed as entity
```